root/layout/index.js

```diff
@@ -88,6 +89,28 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user.birth_date &&
+      moment($c.user.birth_date).dayOfYear() === moment().dayOfYear() &&
+      !getRequestCookie($c.req, 'birthday_message_dismissed_mtime') &&
+      <div className="banner birthday-message">
+        <p>
+          <span aria-label="cakes" role="img">
+            {String.fromCodePoint(0x1F382)}
+            {String.fromCodePoint(0x1F382)}
+            {String.fromCodePoint(0x1F382)}
+          </span>
+          {' '}
+          {l('Happy birthday, and thanks for contributing to MusicBrainz!')}
```

Comment:
Not everyone appreciates automated birthday message, I would suggest an opt-in instead. Plus, birthday is quite unrelated to contributing to MusicBrainz, user registration anniversary or day after the 10..0th edit would probably make more sense.

Reply:
Heh. Now I'm not sure which is stranger: someone upset because we didn't wish them a happy birthday, or someone upset because we did?

I'd really prefer not adding an option for this, and just deciding whether to use their real birthday or account birthday. I see no problem with either.

Reply:
I specifically did a banner, and not an email or something, because I fully understand people don't like to get pointless emails. But I feel a small banner makes sense. I would *also* add one for the account birthday, since thanking people twice a year is nicer than thanking them once! :P